builtin/providers/google/resource_google_project.go

```diff
@@ -58,81 +70,141 @@ func resourceGoogleProject() *schema.Resource {
 func resourceGoogleProjectCreate(d *schema.ResourceData, meta interface{}) error {
 	config := meta.(*Config)
 
-	project, err := getProject(d, config)
+	project := &cloudresourcemanager.Project{
+		Name:      d.Get("name").(string),
+		ProjectId: d.Get("project_id").(string),
+		Parent: &cloudresourcemanager.ResourceId{
+			Id:   d.Get("org_id").(string),
+			Type: "organization",
+		},
+	}
+	op, err := config.clientResourceManager.Projects.Create(project).Do()
 	if err != nil {
-		return err
+		return fmt.Errorf("Error creating project %s (%s): %s.", project.ProjectId, project.Name, err)
 	}
 
-	d.SetId(project)
-	if err := resourceGoogleProjectRead(d, meta); err != nil {
-		return err
+	// Wait for the operation to complete
+	waitErr := resourceManagerOperationWait(config, op, "project to create")
+	if waitErr != nil {
+		// The resource didn't actually create
+		d.SetId("")
+		return waitErr
+	}
+
+	// Set the resourc ID
+	d.SetId(project.ProjectId)
+
+	// Enable services
+	if _, ok := d.GetOk("services"); ok {
+		esr := newEnableServiceRequest(project.ProjectId)
+		services := resourceServices(d)
+		for _, v := range services {
+			//TODO: do this concurrently
+			sop, err := config.clientServiceMan.Services.Enable(v, esr).Do()
+			if err != nil {
+				return fmt.Errorf("Error enabling service %q for project %q: %v", v, project.ProjectId, err)
+			}
+			// Wait for the operation to complete
+			waitErr := serviceManagementOperationWait(config, sop, "api to enable")
+			if waitErr != nil {
+				return waitErr
+			}
+		}
 	}
 
-	// Apply the IAM policy if it is set
+	// Apply IAM policy
 	if pString, ok := d.GetOk("policy_data"); ok {
 		// The policy string is just a marshaled cloudresourcemanager.Policy.
 		// Unmarshal it to a struct.
-		var policy cloudresourcemanager.Policy
+		var policy *cloudresourcemanager.Policy
 		if err = json.Unmarshal([]byte(pString.(string)), &policy); err != nil {
 			return err
 		}
 
-		// Retrieve existing IAM policy from project. This will be merged
-		// with the policy defined here.
-		// TODO(evanbrown): Add an 'authoritative' flag that allows policy
-		// in manifest to overwrite existing policy.
-		p, err := getProjectIamPolicy(project, config)
-		if err != nil {
-			return err
-		}
-		log.Printf("[DEBUG] Got existing bindings from project: %#v", p.Bindings)
-
-		// Merge the existing policy bindings with those defined in this manifest.
-		p.Bindings = mergeBindings(append(p.Bindings, policy.Bindings...))
-
-		// Apply the merged policy
-		log.Printf("[DEBUG] Setting new policy for project: %#v", p)
-		_, err = config.clientResourceManager.Projects.SetIamPolicy(project,
-			&cloudresourcemanager.SetIamPolicyRequest{Policy: p}).Do()
+		// Apply the policy
+		log.Printf("[DEBUG] Setting policy for project: %#v", policy)
```

Comment:
I mentioned this in an earlier comment, but basically we're assuming this resource creates new projects, vs importing existing ones now. I'm becoming more convinced that we want a `google_project` and `google_imported_project` to differentiate the two.